packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -266,6 +306,21 @@ export const CompoundProtocol = {
     const target = { axelarId, remoteAddress };
     await sendGMPContractCall(target, calls, fee, lca, gmpChain);
   },
+  claimRewards: async (ctx, src) => {
+    const { addresses: a, lca, gmpChain, gmpFee } = ctx;
+
+    const session = makeEVMSession();
+    const compoundRewardsController = session.makeContract(
+      a.compoundRewardsController,
+      CompoundRewardsController,
+    );
+    compoundRewardsController.claim(a.compound, src.remoteAddress, true);
+    const calls = session.finish();
+
+    const axelarId = ctx.axelarIds[src.chainName];
+    const target = { axelarId, remoteAddress: src.remoteAddress };
+    await sendGMPContractCall(target, calls, gmpFee, lca, gmpChain);
+  },
```

Comment:
a few lines down where we have `EVMContext<'compound'>`, that's where the new contracts belong: `EVMContext<'compound' | 'compoundRewardsController'>`

There's evidently not enough docs to make this clear; here's hoping we improve on that.